tools/checkpoint_conversion/convert_t5_checkpoints.py

```diff
@@ -0,0 +1,366 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import math
+import os
+import shutil
+
+import numpy as np
+import transformers
+from absl import app
+from absl import flags
+from checkpoint_conversion_utils import get_md5_checksum
+from keras_core import ops
+
+import keras_nlp
+
+PRESET_MAP = {
+    "t5_small_en": "google/t5-v1_1-small",
```

Comment:
We should update these to point to `t5-base`, etc, if we are going for the original weights.

Reply:
Fixed